mods/win7-uac-dim.wh.cpp

```diff
@@ -0,0 +1,121 @@
+// ==WindhawkMod==
+// @id              win7-uac-dim
+// @name            Windows 7 UAC Dim
```

Comment:
IMO the name makes the impression that the mod is for Win7. How about something like "Win7-style UAC Dim"?